python/ETL/transfer.py

```diff
@@ -0,0 +1,160 @@
+#DATABASE_URL=postgresql://doadmin:l5al4hwte8qmj6x8@db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com:25060/treetracker?ssl=true
+
+import psycopg2
+import argparse
+import datetime
+
+DATABASE_URL="postgresql://doadmin:l5al4hwte8qmj6x8@db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com:25060/treetracker?ssl=true"
```

Comment:
Please don't paste database_url here, it will compromise our db.

Reply:
Oh sorry for that. Didn't realize that's a public one.